va-gov/pages/503.html

```diff
@@ -44,7 +44,7 @@
 <body>
   <div class="container">
     <a href="/">
-      <img width="250" alt="Go to VA.gov" src="{% include "va-gov/includes/logo-350px-base64.html" %}" />
+      <img width="250" alt="Go to VA.gov" src="/img/va-primary-horizontal-black-vector.svg" />
```

Comment:
This is gonna sound wild, but because the nature of this page is that the page itself is the only resource required, we have to import the SVG inline, like -

`src="{% include "assets/img/va-primary-horizontal-black-vector.svg" %}" />`

Hopefully the SVG isn't large to slow the page down.